src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -120,6 +120,8 @@ if (MFEM_DIR)
                             TREAT_INCLUDES_AS_SYSTEM ON
                             EXPORTABLE ON)
         blt_list_append(TO TPL_DEPS ELEMENTS mfem)
+    else()
+        target_include_directories(mfem SYSTEM INTERFACE ${MFEM_INCLUDE_DIRS} )
```

Comment:
@white238 -- please check this one out (and the HDF5 ones).
I don't know why the `MFEM_INCLUDE_DIRS` is not being included via the `find_package(mfem)` import, but this one-liner fixes it.